FirebaseAuth/Sources/Swift/Backend/IdentityToolkitRequest.swift

```diff
@@ -67,9 +68,14 @@ private let kIdentityPlatformStagingAPIHost =
 
     // Automatically set the tenant ID. If the request is initialized before FIRAuth is configured,
     // set tenant ID to nil.
-    tenantID = Auth.auth().tenantID
+    if FirebaseApp.app() == nil {
```

Comment:
Do you know why the request would be initialized before Auth in the first place?